### PR TITLE
DOCSP-23084 add reverse link

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DCOSP-23084-link-to-reverse-v0.9.0/connecting/atlas-to-atlas/#roles)

[JIRA](https://jira.mongodb.org/browse/DOCSP-23084)

### CODE REVIEW

File: source/includes/fact-atlas-roles.rst [hunk 1]
```diff
@@ -1,9 +1,10 @@
 The user specified in the connection string must have the
 :atlasrole:`atlasAdmin` role.
 
-To use ``mongosync`` in the reverse direction, you must `create a
-custom role </atlas/reference/api/custom-roles-create-a-role/>`__ that
-grants the following ActionTypes:
+To use ``mongosync`` in the :ref:`reverse direction <c2c-api-reverse>`,
+you must `create a custom role
+</atlas/reference/api/custom-roles-create-a-role/>`__ that grants the
+following ActionTypes:
```
Comment (jmd-mongo) [inline source/includes/fact-atlas-roles.rst:7]: [nit]

We refer to these as "ActionTypes" here and "user actions" in the following paragraph. Consider using the same term throughout.

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the changes, @davemungo! This one LGTM % a small nit.

Thanks!
Joe

File: source/includes/fact-atlas-roles.rst (hunk 1 above)
Comment (davemungo) [reply source/includes/fact-atlas-roles.rst:7]: Done

**Review by davemungo — COMMENTED** [review]
Thanks @jmd-mongo send on for external review next

**Comment by davemungo** [comment]
@alan-zheng-mdb  During an earlier iteration of the page you asked for a link to the /reverse endpoint. This update adds the link now that the API ref is available. Would you have a look?

**Review by alan-zheng-mdb — APPROVED** [review]
LGTM

**Pull request merged**